packages/components-js/tests/e2e/puppeteer/specs/tabs.e2e.ts

```diff
@@ -247,6 +248,39 @@ describe('keyboard', () => {
   });
 });
 
+describe('select Tab Item content', () => {
+  it('should select content of tabs item', async () => {
+    await initTabs();
+    const rect = await page.evaluate(() => {
```

Comment:
Why not make this a local utility within the describe block that can be used in both tests?